orion/sites/admin/src/achieve/achieve.js

```diff
@@ -2,13 +2,54 @@
 require('./achieve.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
 
 export class AchievePage extends React.Component {
+	constructor(props) {
+	    super(props);
+	    this.state = {
+	      list: []
+	    };
+ 	}
+
 	render() {
+		var numAchievements = 5;
+		var fakeAchieve = {
+			year: 2020,
+			message: "Hi"
+		};
+		var fakeAchieve2 = {
+			year: 2019,
+			message: "Hello"
+		};
 		return (
-      <div id="view-achieve">
-        <h1>All Achievements</h1>
-      </div>
+      	<div id="view-achieve">
```

Comment:
keep indents appropriate.
If a <div> is a parent, all children elements should be indented. Or this could be your IDE vs. Github formatting issue...

Reply:
If you're using Atom, go to its Preferences > Editor > Scroll down to look for Tab length. and Give it 4.
From here, you may have to delete some spaces between lines and reindent things to achieve the correct format.

Reply:
So would the <h1> All Achievements </h1> and everything else under it be indented, since they're all under view-achieve?

Reply:
Yes, so it should look like:
```
<div id="view-achieve">
    <h1>All Achievements</h1>
    <ul>
        <li></li>
    </ul>
</div>
```

kind of makes sense?